book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author, :rental
```

Comment:
- [ ] Kindly make sure you are consistent in your naming as you may notice `rental` and `rentals` are not the same variable and this can lead to all sorts of issues in your program. Kindly make sure you fix that.

```suggestion
  attr_accessor :title, :author, :rentals
```

Reply:
That was a typo! sorry!